lib/Track.lua

```diff
@@ -274,10 +274,18 @@ function Track:get_shadow_attribute(attribute)
   elseif attribute == "channel" then return track:get_channel()
   elseif attribute == "device"  then return track:get_device()
   elseif attribute == "pair"    then return track:get_pair()
-  elseif attribute == "jf"      then return track:is_jf()
+  elseif attribute == "is_jf"   then return track:get_is_jf()
```

Comment:
similar to the `t.jf` comment - `is_` is the convention for boolean methods. the `get_` is not used with boolean getters.

Reply:
Changed back and modified get_shadowable_attribute check - see 2c04cd3